client/state/site-settings/exporter/actions.js

```diff
@@ -1,30 +1,84 @@
 /**
  * Internal dependencies
  */
+import notices from 'notices';
+import i18n from 'lib/mixins/i18n';
+
 import {
-	TOGGLE_EXPORTER_ADVANCED_SETTINGS,
-	TOGGLE_EXPORTER_SECTION
+	SET_EXPORT_POST_TYPE,
+	REQUEST_START_EXPORT,
+	REPLY_START_EXPORT,
+	FAIL_EXPORT,
+	COMPLETE_EXPORT
 } from 'state/action-types';
 
 /**
- * Toggle the visibility of the Advanced Settings panel
- * @return {Function}        Action object
+ * Sets the post type to export.
+ *
+ * @param  {Object} postType   The name of the post type to use - 'posts', 'pages', 'feedback', or null for all
+ * @return {Object}            Action object
  */
-export function toggleAdvancedSettings() {
+export function setPostType( postType ) {
 	return {
-		type: TOGGLE_EXPORTER_ADVANCED_SETTINGS
+		type: SET_EXPORT_POST_TYPE,
+		postType
 	};
 }
 
 /**
- * Toggles whether a section of the export is enabled.
+ * Sends a request to the server to start an export.
  *
- * @param  {Object} section   The name of the section to toggle - 'posts', 'pages', or 'feedback'
- * @return {Object}           Action object
+ * @return {Function}         Action thunk
  */
-export function toggleSection( section ) {
+export function startExport() {
+	return ( dispatch ) => {
+		dispatch( {
+			type: REQUEST_START_EXPORT
+		} );
+
+		// This will be replaced with an API call to start the export
+		setTimeout( () => {
+			dispatch( replyStartExport() );
+
+			// This will be replaced with polling to check when the export completes
+			setTimeout( () => {
+				dispatch( completeExport( '#', 'testing-2015-01-01.xml' ) );
+				//dispatch( failExport( 'The reason for failure would be displayed here' ) );
```

Comment:
This is just here for convenience - since the API calls are not connected yet, uncommenting this line and commenting out the one above it will demonstrate what happens when an export fails.
This will be removed in #1359